tools/isobuild/compiler-plugin.js

```diff
@@ -91,7 +93,13 @@ export class CompilerPluginProcessor {
     var sourceProcessorsWithSlots = {};
 
     var sourceBatches = _.map(self.unibuilds, function (unibuild) {
+      const { pkg: { name }, arch } = unibuild;
+      const sourceRoot = name
```

Comment:
ah is this one of the places where `sourceRoot` matters? it's either read from a isopack or from a package directory on disk?